java/oracle/README.md

```diff
@@ -10,10 +10,34 @@ Build it:
 
 ```
 $ cd java-8
+$ tar xzf server-jre-8u152-windows-x64.tar.gz
```

Comment:
You don't have to extract the tar.gz, this is done by the `ADD` instruction in the Dockerfile.